src/arch/xtensa/smp/init.c

```diff
@@ -94,13 +94,13 @@ int slave_core_init(struct sof *sof)
 	if (err < 0)
```

Comment:
Also, please specify in this commit that we are actually also renumber the tracepoints in order to make place for new tracepoints.